src/content/advanced/external-dns/index.md

```diff
@@ -0,0 +1,81 @@
+---
+linkTitle: External DNS
+title: Advanced External DNS configuration
+description: How to customize advanced configuration of the External DNS service in your clusters.
+weight: 25
+menu:
+  main:
+    parent: advanced
+last_review_date: 2022-12-06
+user_questions:
+  - How can I customize the External DNS service configuration?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-cabbage
+---
+
+## Integrate External DNS with AWS Route53 with access key
```

Comment:
This headline appears redundant. Normally the h1 headline (which is generated from the `title` attribute) should be enough as the article's headline.